packages/fxa-content-server/app/tests/spec/views/sign_in_password.js

```diff
@@ -97,7 +101,6 @@ describe('views/sign_in_password', () => {
     });
 
     it('disables Glean metrics on pref', () => {
-      view.beforeRender();
```

Comment:
no need for this now since we've moved teh stubbing to before a `view.render` call, which calls `view.beforeRender`.